app/models.py

```diff
@@ -30,8 +41,14 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    framework_id = db.Column(db.BigInteger,
+                             db.ForeignKey('frameworks.id'),
+                             index=True, unique=False, nullable=False)
+
     supplier = db.relationship(Supplier, lazy='joined', innerjoin=True)
 
+    framework = db.relationship(Frameworks, lazy='joined', innerjoin=True)
```

Comment:
Do we have a separate supplier record for each framework? Or can one supplier have services on more than one framework?